workspaces/global-floating-action-button/.changeset/tidy-glasses-jump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@red-hat-developer-hub/backstage-plugin-global-floating-action-button': patch
+---
+
+render fab on document body when the target element is not found
```

Comment:
the changeset says that the fab should render on the document body, but the code would render it on the `main` element. 

is this correct?

Reply:
```document.querySelector('[class^="BackstagePage-root"]') or document.querySelector('main'); ```
decides the target element. If both of them returns null it should render on the document body. 

I overlooked removing this check https://github.com/redhat-developer/rhdh-plugins/pull/652/files#diff-84de97e74333e2bd4b6f22dfa470b675538294ffd8007b90d04ef134ae69c543L83 which prevented the fab button from rendering when the classname was not found

so, ACC to me the changeset is correct